integration/flink/src/main/java/io/openlineage/flink/OpenLineageFlinkJobListener.java

```diff
@@ -82,7 +83,26 @@ public void onJobSubmitted(@Nullable JobClient jobClient, @Nullable Throwable th
   @Override
   public void onJobExecuted(
       @Nullable JobExecutionResult jobExecutionResult, @Nullable Throwable throwable) {
-    log.debug("JobExecutionResult: {}", jobExecutionResult);
+    log.error("JobExecutionResult: {}", jobExecutionResult);
```

Comment:
Consider switching logging level to info.
Merge consecutive logger lines to a single call.